app/models/RsFormMappings.scala

```diff
@@ -175,13 +175,26 @@ object RsFormMappings {
       so.length match {
         case x => if (x <= 8) true else false
       }
+    case "schemeOrganiserFields.companyRegistrationNum" =>
```

Comment:
collapse that one into the previous case please - they have the same condition (length <= 8)

Reply:
Deleted unused code for this.